styles.css

```diff
@@ -304,12 +305,22 @@ margin-top: -15px;
   font-family: 'Montserrat400', sans-serif;
   font-weight: bold;
   font-size: .9em;
+  align-self: flex-start;
 }
 
 .past-activity-time, .past-activity-description {
   color: #CBC9CF;
 }
 
+.color-icon{
+  background-color: white;
+  height: 30px;
+  width: 5px;
+  border-radius: 5px;
+  align-self: flex-end;
+  margin-left: auto;
```

Comment:
What is the difference between these two?

Reply:
The align-self doesn't work unless the margin-left is added. This can be tested to see if there is a cleaner way.